ixmp/cli.py

```diff
@@ -96,6 +95,28 @@ def report(context, config, key):
     print(r.get(key))
 
 
+# ixmp --url "ixmp://ene_ixmp/MESSAGEix-GLOBIOM SSP1/baseline" solve
+@main.command(help='Run scenario solver')
+@click.option('--remove-solution', is_flag=True, default=False,
+              help='Forces removing solution if exists.')
+@click.pass_obj
+def solve(context, remove_solution):
+    """Solve scenario."""
```

Comment:
It would be good to expand this a little, e.g.

"Solve a Scenario and store results on the Platform.

The scenario indicated by --url or --platform/--model/--scenario/--version is loaded, solved, and the solution results are saved on the Platform. If the scenario already has a solution, --remove-solution must be given."

or something like that.

Check that it looks good on the terminal with `ixmp solve --help`.